validate/static/stage.css

```
@@ -0,0 +1,409 @@
+.gaterule .gk {
+  font: 600 12px/1.5 ui-monospace, SFMono-Regular, Menlo, monospace;
+  text-transform: uppercase; letter-spacing: .05em; color: var(--text);
+}
+.gaterule p { margin: 0; font-size: 13px; line-height: 1.6; color: var(--doc-muted); max-width: none; }
+
+/* ── rule book table extras ──────────────────────────────────────────────── */
+.shadow-row td { opacity: .72; }
+.tag {
+  font: 9.5px/1.4 ui-monospace, SFMono-Regular, Menlo, monospace;
+  text-transform: uppercase; letter-spacing: .06em; color: var(--pile-hold);
+  border: 1px solid currentColor; border-radius: 3px; padding: 0 4px; margin-left: 4px;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)